app/views/pages/toilet_unblock_info.html.haml

```diff
@@ -0,0 +1,8 @@
+%h1 Toilet blocked?
+
+%p It's pretty straightforward to unblock a toilet, you'll need:
+
+%ul
+  %li A large bin bag
+  %li A mop
```

Comment:
Do we have/need something in Trello to make sure we fill these in with real content at some point?

Reply:
I've added a card to the backlog for these - https://trello.com/c/OyIiRYa3/146-content-for-static-pages